VIRTUAL_VACATION/vacation.pl

```diff
@@ -207,6 +215,15 @@
 our $custom_noreply_pattern = 0;
 our $noreply_pattern = 'bounce|do-not-reply|facebook|linkedin|list-|myspace|twitter'; 
 
+# Never send vacation mails for the following recipient email addresses.
+# Useful for e.g. aliases pointing to multiple recipients which have vacation active
+# hence an email to the alias should not trigger vacation messages.
+# By default vacation email addresses will be sent for all recipients.
+# default = ''
+# preventing vacation notifications for recipient info@example.org would look like this:
+# our $novacation_pattern = 'info\@example\.org';
+our $novacation_pattern = ''; 
```

Comment:
Good idea!

Nitpicking: could you please use a slightly more readable variable name, maybe $no_vacation_pattern?